README.md

```diff
@@ -1,59 +1,118 @@
+
+Everything will then be replaced with the result: `don\'t panic|`.
+
+`|` indicates the cursor position after the conversion. (You can place it anywhere in the text.)
+
+### Example 2: Auto-pairing of symbols
+![](https://github.com/caasion/typing-transformer-obsidian/blob/docs/docs/auto-pair.gif)
 
-where `|` indicates the cursor position. `dpx|` is the trigger rule, when `dp` is entered and then `x` follows, the conversion is triggered and the entered `dp` will be replaced by `don't panic` and the cursor will be placed at the end of `panic`.
+The trigger is `《`.
 
+The rule auto-pairs Chinese bookmarks and places the cursor in the middle.
 
-In addition to helping us expand abbreviated phrases, this feature can also handle symbol conversions, such as auto-pairing symbols, or turning fullwidth symbols into halfwidth ones, examples:
-- `'《|' -> '《|》'` rule describes the automatic matching of Chinese book marks and placing the cursor in the middle
-- `'。。|' -> '.|'` rule describes the transformation of two Chinese periods into one English period
+explanation
+### Example 3: Transformation of two full width characters into one half width character
+![](https://github.com/caasion/typing-transformer-obsidian/blob/docs/docs/auto-pair%20and%20transformation.gif)
 
-**Refer to more rules in the setting page of Typing Transfomer and have fun converting!**
+This rule works with the one above.
 
-## Configiurable paired symbols insertion to selection
+1. When a `《` is entered, the second rule auto-pairs it.
+2. When another `《` is entered, the first rule will take priority because it matches. 
+3. The auto-pair rule won't do anything yet. This results in `《《|》`.
+4. This is then converted by the first rule, by the first rule, into `<`.
 
-When an area is selected and some symbols are entered, corresponding paired symbols are automatically added to both sides of the selected area.
+**More examples can be found in the settings page of this plugin. Have fun converting!**
 
-The following symbols are currently supported by default.
+## Deletion Rules (tbd)
 
-- `selected` + `·` -> `` `selected` ``
-- `selected` + `￥` -> `$selected$`
-- `selected` + `<`  -> `<selected>`
-- `selected` + `《`  -> `《selected》`
+A conversion rule has the following syntax:
 
-As with the conversion rules, **selection rules can also be created in the Typing Transformer's settings page**. The basic format is:
+```
+'<deletion trigger>' -x '<result>'
+```
+*Anything in angled brackets are replaced*
+
+Deletion rules are essentially the reverse of input conversion rules, but the deletion of a certain character acts as the trigger.
+
+## Selection Rules
+
+The syntax of a selection rule is as follows:
+
+```
+'<trigger (one char)>' -> '<left insert>' + '<right insert>'
+```
+*Anything in angled brackets are replaced*
+
+When text is selected and **one** character is entered, Typing Transformer will help you insert two characters on each side of the selection.

```

Comment:
I pushed commit just now, there is no "two characters" limitation any more for `<left insert>` and `<right insert>` 

it is kind of bug 😹